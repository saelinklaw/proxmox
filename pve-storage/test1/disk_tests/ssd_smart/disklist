sda
sdb
sdc
sdd
sde
