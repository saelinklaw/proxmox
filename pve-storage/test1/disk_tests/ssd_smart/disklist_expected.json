{
    "sda" : {
	"serial" : "000000000000",
	"vendor" : "ATA",
	"rpm" : 0,
	"gpt" : 1,
	"health" : "PASSED",
	"wearout" : "100",
	"osdid" : -1,
	"size" : 512000,
	"type" : "ssd",
	"devpath" : "/dev/sda",
	"model" : "Crucial_CT500MX200SSD1",
	"wwn" : "0x0000000000000000"
    },
    "sdb" : {
	"model" : "INTEL_SSDSC2BB080G6",
	"devpath" : "/dev/sdb",
	"osdid" : -1,
	"type" : "ssd",
	"size" : 512000,
	"wwn" : "0x0000000000000000",
	"gpt" : 1,
	"rpm" : 0,
	"vendor" : "ATA",
	"serial" : "000000000000000000",
	"wearout" : "97",
	"health" : "PASSED"
    },
    "sdc" : {
	"wwn" : "0x0000000000000000",
	"devpath" : "/dev/sdc",
	"model" : "Samsung SSD 850 PRO 512GB",
	"osdid" : -1,
	"type" : "ssd",
	"size" : 512000,
	"wearout" : "99",
	"health" : "PASSED",
	"gpt" : 1,
	"rpm" : 0,
	"vendor" : "ATA",
	"serial" : "000000000000"
    },
    "sdd" : {
	"rpm" : 0,
	"gpt" : 1,
	"serial" : "000000000000",
	"vendor" : "ATA",
	"wearout" : "100",
	"health" : "PASSED",
	"devpath" : "/dev/sdd",
	"model" : "SanDisk SD8SB8U1T001122",
	"size" : 512000,
	"osdid" : -1,
	"type" : "ssd",
	"wwn" : "0x0000000000000000"
    },
    "sde" : {
	"type" : "ssd",
	"osdid" : -1,
	"size" : 512000,
	"model" : "KINGSTON SHFS37A120G",
	"devpath" : "/dev/sde",
	"wwn" : "0x0000000000000000",
	"vendor" : "ATA",
	"serial" : "000000000000",
	"gpt" : 1,
	"rpm" : 0,
	"health" : "PASSED",
	"wearout" : "91"
    }
}
